helm/jupyterhub-home-nfs/templates/deployment.yaml

```diff
@@ -36,6 +36,13 @@ spec:
         volumeMounts:
         - name: home-directories
           mountPath: /export
+        resources:
```

Comment:
This can be set more simply via the `toJson` helm template function (see full list at https://helm.sh/docs/chart_template_guide/function_list/), like https://github.com/2i2c-org/frx-challenges/blob/4feb779a17000731f4ad982c84952bf514e0570d/helm-chart/templates/deployment.yaml#L97. Admins can directly specify the entire `resources` block, so it's consistent across charts.